remote/lib/remote_endpoint.ex

```diff
@@ -0,0 +1,170 @@
+defmodule Membrane.RTC.Engine.Endpoint.Remote do
+  @moduledoc """
+  A Remote Endpoint
+  """
+
+  use Membrane.Bin
+
+  require Membrane.Logger
+
+  alias Membrane.RTC.Engine.Endpoint.Remote
+
+  def_input_pad :input,
+    demand_unit: :buffers,
+    accepted_format: _any,
+    availability: :on_request
+
+  def_output_pad :output,
+    demand_unit: :buffers,
+    accepted_format: _any,
+    availability: :on_request
+
+  def_input_pad :remote_input,
+    demand_unit: :buffers,
+    accepted_format: _any,
+    availability: :on_request
+
+  def_output_pad :remote_output,
+    demand_unit: :buffers,
+    accepted_format: _any,
+    availability: :on_request
+
+  def_options rtc_engine: [
+                spec: pid(),
+                description: "Pid of parent Engine"
+              ],
+              owner: [
+                spec: pid(),
+                description: """
+                Pid of parent all notifications will be send to.
+                These notifications are:
+                * `{:playlist_playable, content_type}`
+                * `{:cleanup, clean_function}`
+                """
+              ],
+              link_proposal: [
+                spec: Remote.LinkProposal.t() | nil,
+                default: nil
+              ]
+
+  @impl true
+  def handle_init(_ctx, opts) do
+    state =
+      opts
+      |> Map.from_struct()
+      |> Map.merge(%{
+        partner_endpoint: nil,
+        status: :initialized
+      })
+
+    {[], state}
+  end
+
+  @impl true
+  def handle_setup(ctx, state) do
+    {:endpoint, endpoint_id} = ctx.name
+    myself = :"#{endpoint_id}"
```

Comment:
Are you sure creating atoms dynamically from UUID is a good idea? Isn't it a memory leak?